scripts/base/frameworks/analyzer/main.zeek

```diff
@@ -88,6 +88,16 @@ export {
 	## Returns: The analyzer name corresponding to the tag.
 	global name: function(tag: Analyzer::Tag) : string;
 
+	## Whether the given analyzer name is valid.
+	##
+	## This can be used before calling :zeek:see:`Analyzer::get_tag` when
+	## name may not actually be a valid analyzer name.
+	##
+	## name: The analyzer name.
+	##
+	## Returns: True if the given name is a valid analyzer, else false.
+	global is_tag: function(name: string): bool;
```

Comment:
Just a tiny question about naming: given that the other function is called `get_tag`, would `has_tag` fit better here?